packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -1878,6 +1939,20 @@ describe('Select', () => {
                 await waitForUpdatesAsync();
                 expect(pageObject.isNoResultsLabelVisible()).toBeTrue();
             });
+
+            it('if all results are hidden, and loadingVisible is true, no results label is not shown', async () => {
+                element.loadingVisible = true;
+                await waitForUpdatesAsync();
+                await pageObject.openAndSetFilterText('abc');
+                expect(pageObject.isNoResultsLabelVisible()).toBeFalse();
+            });
+
+            it('does not automatically filter options when filterMode is manual', async () => {
```

Comment:
You have another test with the same name earlier in the file. Is it a duplicate?

Reply:
It's the same test, just done, additionally, when there are groups present, as that exercises a slightly different code path.

Reply:
No strong opinions about how it's achieved, but they should have different resolved names (the name that appends all the nested `describe` descriptions) so that you can tell which test is failing from a log file.

Reply:
Yep. There is a describe block for "groups" that disambiguates them.